app/assets/stylesheets/local/forms.scss

```diff
@@ -1,62 +1,62 @@
-form.button_to {
-  padding-bottom: 0;
-
-  #proposition-links & {
-    input[type="submit"] {
-      background: transparent;
-      border: none;
-      outline: none;
-      color: $white;
-      padding: 0;
-      margin: 0;
-      font-weight: bold;
-      cursor: pointer;
-      text-decoration-skip: ink;
-
-      &:hover {
-        text-decoration: underline;
-      }
-
-      &:focus {
-        color: $black;
-        background-color: $yellow;
-        outline: 3px solid $yellow;
+form {
+  & .app_heading {
```

Comment:
We don't need the `&` here. As `.app_heading` is a child we can define it without needing to use the parent selector using the `&` symbol

Reply:
Changes addressed in a fixup commit :)